tests/unit/bokeh/test_model.py

```diff
@@ -134,6 +134,52 @@ def test_all_builtin_models_default_constructible() -> None:
             bad.append(name)
         assert bad == []
 
+def test_select() -> None:
+        # we aren't trying to replace test_query here, only test
```

Comment:
Indentation is off.

Reply:
That was silly of me to not notice the obvious.